### PR TITLE
Avoid ThreadLocalRandom in BenchmarkDecimalAggregation

### PR DESCRIPTION
**Pull request by raunaqmorarka**
## Description

Makes the benchmark more stable by using the same random seed 
Also includes some nulls in the input

## Release notes

(x) This is not user-visible or docs only and no release notes are required.
( ) Release notes are required, please propose a release note for me.
( ) Release notes are required, with the following suggested text:

### CODE REVIEW

File: core/trino-main/src/test/java/io/trino/operator/aggregation/BenchmarkDecimalAggregation.java [hunk 1]
```diff
@@ -150,17 +152,13 @@ private Block createIntermediateValues(GroupedAggregator aggregator, GroupByIdBl
...
             TestingAggregationFunction implementation = functionResolution.getAggregateFunction(QualifiedName.of(function), fromTypes(type));
             partialAggregatorFactory = implementation.createAggregatorFactory(PARTIAL, ImmutableList.of(0), OptionalInt.empty());
             finalAggregatorFactory = implementation.createAggregatorFactory(FINAL, ImmutableList.of(0), OptionalInt.empty());
 
-            BlockBuilder builder = type.createBlockBuilder(null, ELEMENT_COUNT);
-            for (int i = 0; i < ELEMENT_COUNT; i++) {
-                writer.write(builder, i);
-            }
-            return new Page(builder.build());
+            return new Page(createRandomBlockForType(type, ELEMENT_COUNT, 0.03f));
```
Comment (sopel39) [inline core/trino-main/src/test/java/io/trino/operator/aggregation/BenchmarkDecimalAggregation.java:161]: what is 0.03f here?

Comment (lukasz-stec) [inline core/trino-main/src/test/java/io/trino/operator/aggregation/BenchmarkDecimalAggregation.java:161]: does it make sense to have `nullRate` benchmark param to support both nullable and non nullable cases?

**Review by sopel39 — APPROVED** [review]

**Review by lukasz-stec — APPROVED** [review]
lgtm

File: core/trino-main/src/test/java/io/trino/operator/aggregation/BenchmarkDecimalAggregation.java (hunk 1 above)
Comment (raunaqmorarka) [reply core/trino-main/src/test/java/io/trino/operator/aggregation/BenchmarkDecimalAggregation.java:161]: nulls fraction

Comment (raunaqmorarka) [reply core/trino-main/src/test/java/io/trino/operator/aggregation/BenchmarkDecimalAggregation.java]: added nullRate param

**Review by lukasz-stec — APPROVED** [review]
lgtm

**Pull request merged**